tests/ui/circulation/test_inhouse_cipo.py

```diff
@@ -0,0 +1,86 @@
+# -*- coding: utf-8 -*-
+#
+# RERO ILS
+# Copyright (C) 2020 RERO
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published by
+# the Free Software Foundation, version 3 of the License.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+"""In house circulation policy tests."""
+
+from copy import deepcopy
+from datetime import datetime
+
+import ciso8601
+import pytz
+
+from rero_ils.modules.items.api import Item
+from rero_ils.modules.items.models import ItemStatus
+from rero_ils.modules.libraries.api import Library
+from rero_ils.modules.loans.api import Loan, LoanAction, LoanState
+
+
+def test_less_than_one_day_checkout(
+        circ_policy_less_than_one_day_martigny,
+        patron_martigny_no_email,
+        patron2_martigny_no_email,
+        item_lib_martigny,
+        loc_public_martigny,
+        librarian_martigny_no_email,
+        item_on_shelf_martigny_patron_and_loan_pending):
+    """Test checkout on an ON_SHELF item with 'less than one day' cipo."""
+    # Create a new item in ON_SHELF (without Loan)
+    data = deepcopy(item_lib_martigny)
+    data.pop('barcode')
+    data.setdefault('status', ItemStatus.ON_SHELF)
+    created_item = Item.create(
+        data=data, dbcommit=True, reindex=True, delete_pid=True)
+
+    # Check item is ON_SHELF and NO PENDING loan exist!
```

Comment:
I do not think you need to retest this because you just created the item and it has no requests for sure. you can safely remove lines 49-> 52

Reply:
Done